notebook/agentchat_qdrant_RetrieveChat.ipynb

```diff
@@ -21,7 +21,7 @@
     "Some extra dependencies are needed for this notebook, which can be installed via pip:\n",
     "\n",
     "```bash\n",
-    "pip install \"pyautogen[retrievechat]>=0.2.3\" \"flaml[automl]\" \"qdrant_client[fastembed]\"\n",
+    "pip install \"pyautogen[retrievechat-qdrant]\"\n",
```

Comment:
flaml[automl] should be kept as a standalone package needed for this notebook only.